schedule/yast/textmode/textmode@svirt-xen.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name: textmode@svirt-xen-pv
```

Comment:
perhaps this one could be unified with the general one via VIRSH_VMM_FAMILY, wdty?

Reply:
There are conflicts. I tried and i realize that it simpler separated. AFAIK this is preferable when the scheduling starts becomes complicated. i am resolving the comment. feel free to reopen it for discussion

Reply:
could you list the conflicts? They look really similar. It looks like it would be just to add another conditional section (due to there is not differences among different virtualization in this case)

Reply:
if i remember correctly they condition section interferes with the ppc64le as it is also has `svirt` BACKEND.

Reply:
that is correct, thanks for clarify, I guess we would have needed to add a new variable at machine level for s390x to avoid that ambiguity in our schedules with svirt.

Reply:
could you please correct s/name: textmode@svirt-xen-pv/name: textmode@svirt-xen/

Reply:
done